### ISSUE TITLE
Think about feature modules and the possibility to break down the app into distinct modules

### ISSUE THREAD

**Issue by subotic** [dhlab-basel/Salsah#217]
- This would be a pre-requirement for lazy loading, i.e. faster page load times.
- https://angular.io/guide/feature-modules

**Comment by kilchenmann** [comment]
This is what we're doing on [salsah-modules](https://github.com/dhlab-basel/salsah-modules)!?

**Comment by subotic** [comment]
> This is what we're doing on salsah-modules!?

I don't think that this as anything to do with `salsah-module`. Here I'm talking about organizing the `Salsah` app in so-called `feature-modules`, so that lazy loading can be possible.

**Comment by kilchenmann** [comment]
But it's really similar... But we can try feature-modules in the dashboards
